gulpfile.ts

```diff
@@ -55,6 +56,10 @@ gulp.task('run_server', () => {
   });
 });
 
+gulp.task('clean', (callback) => {
```

Comment:
I think callback argument is probably unnecessary in this case.

Reply:
Okay. I remove callback argument in this code.
Thank you for review :D